dm/tests/extend_column/conf/dm-task.yaml

```diff
@@ -0,0 +1,73 @@
+---
+name: test
+task-mode: all
+is-sharding: false
+meta-schema: "dm_meta"
+# enable-heartbeat: true
```

Comment:
```suggestion

```
heartbeat is also deprecated, please also delete heartbeat-update-interval and heartbeat-report-interva